### ISSUE TITLE
Ohai plugins in the /ohai directory are not displayed in the cookbook view

### ISSUE THREAD

**Issue by tas50** [chef/automate#5097]
## Describe the bug
In the latest releases of Chef Infra Server 12 we added the ability to upload cookbooks with a /ohai directory containing plugins. These plugins are automatically installed and this is our preferred method of installing ohai plugins now. We need to make sure these display in Automate, but they currently do not.

## To Reproduce
- Upload a cookbook with a `/ohai/whatever.rb` file
- From the automate UI
  - Select infrastructure
  - Select Chef Infra Servers
  - Select the server
  - Select the org
  - Select the cookbook tab
  - Select that cookbook

## Expected behavior
You should see  an ohai directory with the plugin

## Screenshots
![Screen Shot 2021-05-24 at 9 49 01 AM](https://user-images.githubusercontent.com/1015200/119380233-4d5bf600-bc75-11eb-88e9-866139035783.png)

## Versions (please complete the following information):
- OS: macOS
- Browser Firefox
- Automate Build Number 20210518101414

**Comment by kalroy** [comment]
Point to note:
Ohai folder is not being shown in Manage too.

Question:
Is Chef Server API returning /ohai content?

This is a new feature which needs triaged.

**Comment by tas50** [comment]
Yeah manage has a pretty flawed view of cookbooks as well. When manage was created we had set directories that could be part of a cookbook. We no longer have that so we need to display the cookbooks differently. If there's bad data coming back from the server API @kalroy it would be good to know so we can update that.
